keps/sig-node/693-topology-manager/README.md

```diff
@@ -594,49 +593,305 @@ feature gate and set the CPU Manager policy to 'static'.
 
-* [AutoNUMA][numa-challenges]: This kernel feature affects memory
-  allocation and thread scheduling, but does not address device locality.
+#### Beta (v1.20) [COMPLETED]
 
-# References
+* Support pod-level resource alignment.
+
+#### GA (stable) [COMPLETED]
+
+- N examples of real-world usage
+- N installs
+- More rigorous forms of testing.
+- Allowing time for user feedback.
+- Add support for device-specific topology constraints beyond NUMA.
+- Support hugepages alignment.
+
+#### Deprecation
+
+- Announce deprecation and support policy of the existing flag
+- Two versions passed since introducing the functionality that deprecates the flag (to address version skew)
+- Address feedback on usage/changed behavior, provided on GitHub issues
+- Deprecate the flag
+
+### Upgrade / Downgrade Strategy
+
+Not Applicable.
+
+### Version Skew Strategy
+
+This feature is kubelet specific, so version skew strategy is N/A.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: TopologyManager
+  - Components depending on the feature gate: Topology Manager
+
+Kubelet Flag for the Topology Manager Policy, which is described above. The `none` policy will be the default policy.
+ 
+- Proposed Policy Flag:  
+ `--topology-manager-policy=none|best-effort|restricted|single-numa-node`  
+
+Based on the policy chosen, the following flag will determine the scope with which the policy is applied (i.e. either on a pod-by-pod or a container-by-container basis). The `container` scope will be the default scope.
+
+- Proposed Scope Flag:  
+ `--topology-manager-scope=container|pod`  
+ 
+
+###### Does enabling the feature change any default behavior?
+
+If just the feature gate is enabled, there is no change in behavior as Topology Manager policy
+defaults to `none` policy. In this case, kubelet does not consult Topology Manager and does not influence
+the placement decision (which is the defult behavior).
+
+If Topology Manager is configured with `single-numa-node` or `restricted` policy, the admission flow
+changes for a pod where at least two of the following is true:
+
+* Node is using the static CPU manager policy (if true, implies pod is Guaranteed QoS )
+* Pod consumes some device A that exports locality hints
+* Pod consumes some device B that exports locality hints
+* Node is using the static Memory manager policy (if true, implies pod is Guaranteed QoS )
+* Pod consuming pre-allocated hugepages
+
+Topology Manager takes into account hints received from hint providers like CPU Manager,
+Memory Manager and Device Manager to either admit a pod to the node or reject it.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, this feature can be disabled by specifying `TopologyManager` feature gate
```

Comment:
Presumably the feature gate is locked on in `stable`? So we should just mention that. I am not sure if we do that in general, but I think we should. Just state: "Since going to `stable` in 1.27, the feature gate is locked on as is the standard practice in Kubernetes." You can leave all the rest too, though, since people will not be running 1.27 for a long time.

Reply:
Sure, will update.